### ISSUE TITLE
Errors in markup are always at 0:0

### ISSUE THREAD

**Issue by bookpauk** [sveltejs/eslint-plugin-svelte#1247]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.

### What version of ESLint are you using?

9.28.0

### What version of `eslint-plugin-svelte` are you using?

3.9.2

### What did you do?

<details>
<summary>Configuration</summary>

```
import js from '@eslint/js';
import svelte from 'eslint-plugin-svelte';
import globals from 'globals';

/** @type {import('eslint').Linter.Config[]} */
export default [
    js.configs.recommended,
    ...svelte.configs.recommended,
    {
        languageOptions: {
            globals: { ...globals.browser, ...globals.node }
        }
    },
    {
        files: ['**/*.svelte', '**/*.svelte.js'],
    }
];
```
</details>

```svelte
<div>
    </a>
</div>
```

### What did you expect to happen?

expected eslint output:
  2:4  error  Parsing error: `</a>` attempted to close an element that was not open
https://svelte.dev/e/element_invalid_closing_tag

### What actually happened?

actual eslint output:
  0:0  error  Parsing error: `</a>` attempted to close an element that was not open
https://svelte.dev/e/element_invalid_closing_tag

### Link to **GitHub Repo** with Minimal Reproducible Example

no repo
